staff/fede-gomez/cv/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CV from Fede Gomez</title>
+</head>
+<body>
+    <header>
+        <h1 id="title">
+             Fede Gómez CV
+        </h1>
+        <p>Coder en Skylab Coders Academy</p>
+        <address>
+            <p>Federico Gómez Cucchiararo</p>
+            <p>Calle 20, 08210 Barcelona</p>
+            <a href="mailto:fedegomezcucchiararo@gmail">fedegomezcucchiararo@gmail.com</a>
+        </address>
+    </header>
+    <section id="education">
```

Comment:
Nice work!
You can use ASIDE for write inside the SECTION's

Reply:
That is really helpful. Thanks!!

Reply:
that's NOT RIGHT, do not use ASIDE for non-understood reason. please, re-RTFM: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside